data/config_options.yml

```
@@ -978,7 +978,7 @@ options:
     description: |
       Whether to skip sending request parameters to AppSignal.
 
-      For more information please read about [send_params](parameter-filtering.html#filter-all-parameters) in filtering request parameters.
+      For more information please read about [send_params](parameter-filtering.html) in filtering request parameters.
```

This link looks like it relies on redirects from `/configuration/parameter-filtering.html` to `/application/parameter-filtering.html`. We can also link to it directly.

```suggestion
      For more information please read about [send_params](/application/parameter-filtering.html) in filtering request parameters.
```